tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+How to Set Up WPA Enterprise (EAP-TTLS-PAP) authentication
+==========================================================
+
+In this tutorial, we will guide you set up WPA Enterprise (EAP-TTLS-PAP)
+authentication for WiFi networks using OpenWISP. The RADIUS capabilities
+of OpenWISP provides integration with FreeRADIUS to allow users to
+authenticate with their django user accounts. Users can either be created
+manually via the admin interface, generated or imported from CSV.
+
+You can try this feature on our :doc:`OpenWISP Demo System <./demo>`!
+
+Pre-requisites for following this tutorial
+------------------------------------------
+
+Enabled OpenWISP RADIUS
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    You don't need to do anything if you are following this tutorial on
+    the :doc:`OpenWisp Demo System <./demo>`. OpenWISP RADIUS is already
+    enabled on the :doc:`OpenWISP Demo System <./demo>`.
```

Comment:
```suggestion
    You don't need to do anything if you are following this tutorial on
    the :doc:`OpenWisp Demo System <./demo>` because
    OpenWISP RADIUS is already enabled there.
```